scripts/src/updateApplicationStatus.ts

```diff
@@ -0,0 +1,79 @@
+import config from "@api/config";
+import { MongoClient, ObjectId } from "mongodb";
+
+// Throw and show a stack trace on an unhandled Promise rejection instead of logging an unhelpful warning
+process.on("unhandledRejection", err => {
+  throw err;
+});
+
+const client = new MongoClient(config.database.mongo.uri);
+
+const updateApplication = async (
+  acceptedApplications: string[], // List of Application Ids
+  waitlistedApplications: string[], // List of Application Ids
+  confirmationBranchId: string,
+  travelType: string
+) => {
+  await client.connect();
+  const db = client.db("registration");
+  const collection = db.collection<any>("applications");
+
+  // Update accepted applications
+  collection.updateMany(
+    {
```

Comment:
Can you also update these updates to get the number of applications updated? Then, at the end of the script you can output like "x applications accepted" and "x applications waitlisted"